Documentation/TypoScript/Index.rst

```diff
@@ -0,0 +1,170 @@
+:navigation-title: TypoScript introduction
+..  include:: /Includes.rst.txt
+
+..  _typoscript:
+
+=========================================
+A minimal page created by pure TypoScript
+=========================================
+
+TypoScript is the basic configuration script used to configure the frontend
+output of a page in TYPO3.
+
+You can find detailed information about
+:ref:`TypoScript <t3coreapi:typoscript-syntax-syntax>` and its syntax in
+the reference "TYPO3 explained".
+
+The semantics of TypoScript settings are listed in the
+:ref:`TypoScript Reference <t3tsref:start>`.
+
+..  _typoscript-hello-world:
+
+"Hello world" example in TypoScript
+===================================
+
+..  versionchanged:: 13.1
+    The site set can be used as
+    :ref:`TypoScript provider <t3coreapi:site-sets-typoscript>`. It is not
+    necessary to use a TypoScript record as is explained in tutorials for
+    TYPO3 versions 12,4 and below.
+
+Put the following TypoScript in a file called :file:`setup.typoscript` within
+your :ref:`Site set <t3coreapi:site-sets>`. The site set is the folder
+containing your :ref:`site configuration <site-configuration>`.
+
+..  literalinclude:: _minimal.typoscript
+    :caption: config/sites/main/setup.typoscript
+    :linenos:
+
+#.  Is a comment. See the Syntax of comments in TypoScript:
+    :ref:`Comments <t3coreapi:typoscript-syntax-syntax-comment-blocks>`.
+#.  Assigns a top level object of type :ref:`PAGE <t3tsref:object-type-page>`
+    to a variable called `page`.
+#.  The `page` gets more options in this block. See the
+    :ref:`Blocks in the TypoScript syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>`.
+#.  Another comment.
+#.  Assigns a content object (also called "cObject") of type
+    :ref:`TEXT <t3tsref:cobj-text>` to
+    :confval:`index number 10 <t3tsref:page-array>` of `page`. It has the path
+    `page.10` if you want to change it later.
+#.  Assigns the value `Hello, world.` to the `value` property of the TEXT
+    cObject stored in path `page.10`.
+
+Clear all caches via the following console command or the button in the backend:
```

Comment:
As this is the Getting Started tutorial: "or the button in the backend" - which of the many buttons?

Reply:
I added a todo as that page is still missing